osu.Game.Rulesets.Mania/Objects/Drawables/DrawableHoldNote.cs

```diff
@@ -322,10 +325,6 @@ public void OnReleased(KeyBindingReleaseEvent<ManiaAction> e)
             if (e.Action != Action.Value)
                 return;
 
-            // do not run any of this logic when rewinding, as it inverts order of presses/releases.
-            if (Time.Elapsed < 0)
-                return;
```

Comment:
I think we still want this guard to be here. In debugging the end of this method is being reached when rewinding autoplay which seems bad...

Reply:
Have added this back.